.github/workflows/ubuntu.yml

```
@@ -29,28 +34,29 @@ jobs:
     strategy:
         fail-fast: false
         matrix:
-          psql: [13,14,15,16]
+          psql: [13,14,15,16,17,18]
```

YAMLlint (1.37.1)

[error] too few spaces after comma

(commas)